apps/api/src/app/shared/shared.module.ts

```diff
@@ -85,7 +74,6 @@ const DAL_MODELS = [
   WorkflowOverrideRepository,
   ControlValuesRepository,
   PreferencesRepository,
-  ...getDynamicAuthProviders(),
```

Comment:
we don't need to import these into `SharedModule` because its already imported in `AuthModule` (which is part of `AppModule` same as `SharedModule`)

Reply:
I reverted this part - reason in the comment in code